src/components/shared/flight/FlightCard/FlightCard.tsx

```diff
@@ -1,22 +1,30 @@
-import React, { FC } from 'react';
-import { Heading, Stat, StatNumber, StatHelpText, Flex } from '@chakra-ui/react';
+import { FC } from 'react';
+import { Heading, Stat, StatNumber, StatHelpText, Flex, Skeleton, StatLabel } from '@chakra-ui/react';
 import { format } from 'date-fns';
 import { Flight } from '@/models/Flight';
+import { useTranslation } from 'next-i18next';
 
 interface IFlightCardProps {
 	flight: Flight;
 }
 
-export const FlightCard: FC<IFlightCardProps> = ({ flight, ...rest }) => (
-	<Flex align="center" p={5} borderWidth="1px" shadow="md" {...rest}>
-		<Heading as="h2" w="50%" mb={2} size="lg">
-			{flight.name}
-		</Heading>
-		<Stat w="50%">
-			<StatNumber>${flight.currentSeatPrice}</StatNumber>
-			<StatHelpText>
-				{format(new Date(flight.departsAt), 'MMM dd')} - {format(new Date(flight.arrivesAt), 'MMM dd')}
-			</StatHelpText>
-		</Stat>
-	</Flex>
-);
+export const FlightCard: FC<IFlightCardProps> = ({ flight, ...rest }) => {
+	const { t } = useTranslation('flight-card');
+
+	return (
+		<Flex align="center" p={5} borderWidth="1px" {...rest}>
+			<Heading as="h2" w="50%" mb={2} size="lg">
+				{flight.name}
+			</Heading>
+			<Stat w="50%">
+				<StatLabel srOnly>{t('seatPriceLabel')}</StatLabel>
```

Comment:
screen reader only. It will be in the a11y tree but it won't be visible on the screen. 

https://tailwindcss.com/docs/screen-readers 

https://github.com/chakra-ui/chakra-ui/blob/f5b1a3569fd0c654897d2397b3d0bd4677783fa7/packages/core/styled-system/src/config/others.ts#L5